examples/purchase-tester/src/main/res/navigation/nav_graph.xml

```diff
@@ -29,5 +29,14 @@
                 android:name="offering"
                 app:argType="com.revenuecat.purchases.Offering" />
     </fragment>
+    <fragment
+            android:id="@+id/configureFragment"
+            android:name="com.revenuecat.purchasetester.ConfigureFragment"
+            android:label="ConfigureFragment" >
+        <action
+                android:id="@+id/action_configureFragment_to_loginFragment"
+                app:destination="@id/loginFragment"
+                app:popUpTo="@id/nav_graph.xml" />
```

Comment:
I added the `popUpTo` so we can't go back to the first screen by pressing the back button. Right now, If we want to  go back to the configuration fragment we need to restart the app. I plan to add a mechanism to go back from the login and overview fragments in future PRs.